### PR TITLE
Addition of Configuration

### PR DESCRIPTION
**Pull request by joinimran**
Updated a configuration in the document to reflect the CPU usage during scheduled scans.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8929

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by illfated — APPROVED** [review]
Looks good to me. Adheres to the existing document codestyle.

File: windows/security/threat-protection/microsoft-defender-atp/linux-preferences.md [hunk 1]
```diff
@@ -245,6 +245,16 @@ Specify the maximum number of entries to keep in the scan history. Entries inclu
 | **Possible values** | 10000 (default). Allowed values are from 5000 items to 15000 items. |
 | **Comments** | Available in Defender for Endpoint version 101.04.76 or higher. |
 
+#### Maximum on-demand scan threads
+
+Specify the maximum number CPU used during scheduled scans.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/linux-preferences.md:250]:
```suggestion
Specify the maximum number of CPUs used during scheduled scans.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**